conda/recipes/libcuspatial/build.sh

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+
+# This assumes the script is executed from the root of the repo directory
+export CUDF_HOME="${WORKSPACE}/cudf"
+# show environment
+printenv
+# Cleanup local git
+git clean -xdf
+# build cudf with verbose output
+cd $CUDF_HOME
+./build.sh -v libnvstrings nvstrings libcudf cudf
```

Comment:
Does `libcuspatial` need `nvstrings` and `cudf` and not just the lib ones?

Reply:
Deferring to @harrism

Reply:
It needs cudf. It doesn't use nvstrings itself but cudf probably depends on it.

Reply:
Shouldn't it only need `libnvstrings` and `libcudf`?

Reply:
It needs cudf bindings in the Python API, I assume.